hasher-matcher-actioner/src/OpenMediaMatch/blueprints/hashing.py

```diff
@@ -105,22 +113,30 @@ def hash_media() -> dict[str, str]:
     Output:
         * Mapping of signal types to hash values. Signal types are derived from the content type of the provided URL
     """
-    media_url = request.args.get("url", None)
-    if media_url is None:
-        abort(400, "Missing required parameter: url")
-
     try:
-        return hash_url_content(media_url)
+        result = hash_url_content(
+            query.url,
+            content_type_hint=query.content_type,
+            signal_type_names=query.types,
+        )
+        return HashResponse(**result)
     except ValueError as e:
         abort(400, str(e))
```

Comment:
nit: this hash response should probably be outside the try
```suggestion
    except ValueError as e:
        abort(400, str(e))
   return HashResponse(**result)
```